dataset/dictionary/characters.json5

```
@@ -4607,7 +4666,7 @@
     zhCN: "阿蕾奇诺",
     notes: "[召使](/the-knave/)の通称",
     notesZh: "[仆人](/the-knave/)的通称",
-    tags: [ "snezhnaya", "character-sub" ],
+    tags: [ "snezhnaya", "fontaine", "character-main" ],
```

Can you also add `enemy-boss` here?

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss" ],
```